src/main/java/kitchenpos/order/event/TableGroupEvent.java

```diff
@@ -0,0 +1,22 @@
+package kitchenpos.order.event;
+
+import java.util.List;
+
+public class TableGroupEvent {
+
+    Long tableGroupId;
+    List<Long> orderTableIds;
```

Comment:
```suggestion
    private final Long tableGroupId;
    private final List<Long> orderTableIds;
```

접근제어자를 생략한 것은 의도된 내용일까요?

Reply:
실수가 많네요 좀더 신경 쓰겠습니다